client/components/payment-logo/index.jsx

```diff
@@ -7,20 +7,43 @@
 import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
+import { keys } from 'lodash';
+
+const ALT_TEXT = {
+	alipay: 'Alipay',
+	amex: 'American Express',
+	bancontact: 'Bancontact',
+	diners: 'Diners Club',
+	discover: 'Discover',
+	eps: 'eps',
+	giropay: 'Giropay',
+	ideal: 'iDEAL',
+	jcb: 'JCB',
+	mastercard: 'MasterCard',
+	p24: 'Przelewy24',
+	paypal: 'PayPal',
+	placeholder: '',
+	unionpay: 'UnionPay',
+	visa: 'VISA',
+};
+
+const POSSIBLE_TYPES = keys( ALT_TEXT );
 
 class PaymentLogo extends React.Component {
 	static propTypes = {
-		type: PropTypes.string.isRequired,
-		altText: PropTypes.string.isRequired,
+		type: PropTypes.oneOf( POSSIBLE_TYPES ),
+		altText: PropTypes.string,
 		isCompact: PropTypes.bool,
 	};
 
 	render() {
-		const classes = classNames( 'payment-logo', `is-${ this.props.type }`, {
-			'is-compact': this.props.isCompact,
+		const { altText, isCompact, type } = this.props;
+
+		const classes = classNames( 'payment-logo', `is-${ type }`, {
```

Comment:
It might be nice to add support for adding in `this.props.className` here. We'd probably want to add `className` to our `propTypes` if we do, though.

Reply:
Thank you! Added in commit 0d4661373e7d40298a1c5adde062cc1cf25421c2 as suggested 😄